src/modules/display.js

```diff
@@ -0,0 +1,24 @@
+import { getData } from './postGet.js';
+
+const error = document.querySelector('.error');
+
+export const display = (data) => {
+  const results = document.getElementById('results');
+  results.innerHTML = '';
+
+  data.forEach(({ user, score }) => {
```

Comment:
[OPTIONAL]

As the name implies 'Leaderboard', I will strongly recommend you sort the scores so that the highest scores move to the top which the lowest scores at the bottom.

you can use the sort method in javascript to achieve this 😄